pkgs/development/python-modules/fonttools/default.nix

```diff
@@ -72,6 +73,7 @@ buildPythonPackage rec {
     "Tests/ufoLib"
   ];
 
+  nativeBuildInputs = [ setuptools-scm ];
 
```

Comment:
pythonImportsCheck missing and nativeBuildInputs goes after src and before checkInputs.